src/Toolchain/ToolchainEnv.ts

```diff
@@ -119,10 +134,13 @@ class ToolchainEnv extends Env {
 
   install(toolchain: Toolchain) {
     if (this.installed !== undefined) {
-      throw Error('Before installing, uninstall toolchain');
+      Balloon.error(
+          'Debian toolchain is allowed to install only one. Please uninstall toolchain first.');
```

Comment:
```suggestion
          'The toolchain installed in your system must be one and only. Please uninstall the existing toolchain first.');
```
(optional) or one->once.